_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1631,6 +1665,394 @@ cd infrastructure-live/root/_global/cloudtrail
 terragrunt apply
 ----
 
+=== Enable AWS Config in the root account
```

Comment:
Rob, just took a quick glance and noticed you are updating the guide to deploy each module, one at a time, but we should instead replace the deployment section entirely with a recommendation to use the new account-baseline modules! That would reduce the entire guide down to solely deploying the three account-baseline modules, turning on/off whatever settings the customer wants.

Reply:
@brikis98 I would have done but according to your PRD it wasn't a "must-have".

Reply:
It wasn't a must-have because the account-baseline modules weren't must-haves... But since we did them, it only makes sense to update the guide to use them!

Reply:
@brikis98 okay I'm really eager to try and finish this, this week, which means today, so I would appreciate a further review of the rest of the stuff I've done so far before I start making changes.

Reply:
+1 on Jim's suggestion to use the account-baseline modules. In fact, I show those in most of my sales demos these days and customers always respond very favorably to them. It will also present the setup process as less complex, which is a key driver for customers.

Reply:
I've updated the guide to use the new `account-baseline` modules.